Cargo.toml

```diff
@@ -8,3 +8,126 @@ members = [
   "runtime/mainnet",
   "runtime/devnet",
 ]
+
```

Comment:
We can switch to `resolver  = "2"` here as part of the update. It's a better resolver, and the default (resolver v1) issues a cargo warning when building. 
```toml
resolver = "2"
```

Reply:
Done